lib/timeline/component/ItemSet.js

```diff
@@ -1920,6 +1920,7 @@ class ItemSet extends Component {
    */
   _onGroupClick(event) {
     const group = this.groupFromTarget(event);
+    this.body.emitter.emit('toggle-group', { group });
```

Comment:
Naming convention between the new different events may be more coherent.
In the other commit: "groupExpanded", "groupCollapsed".
I personally prefer  the passive form for event. Instead of "toggle-group" (that could be also a name instead of an action), I would prefer "groupToggled" to name the event.